cupy/core/_routines_math.pyx

```diff
@@ -84,6 +84,24 @@ cdef ndarray _ndarray_cumprod(ndarray self, axis, dtype, out):
     return cupy.cumprod(self, axis, dtype, out)
 
 
+cdef ndarray _ndarray_nansum(ndarray self, axis, dtype, out, keepdims):
+    if numpy.iscomplexobj(self):
```

Comment:
`numpy.iscomplexobj(cupy.ndarray)` seems not to work well in the old version's NumPy. Could you fix as follows?
```suggestion
    if self.dtype.kind == 'c':
```